Client/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App.jsx'
+
+
+ReactDOM.render(<App />, document.getElementById('root'));
+
+
```

Comment:
There should only be one extra line after the end of the last line of code in a file